<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>897</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>220</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell Condensed</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(139, 125, 123);
border-style: outset;
border-radius:5px; 
border-width: 5px;
border-color:rgb(149, 145, 143);
</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>220</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(139, 125, 123);
border-style: outset;
border-radius:5px; 
border-width: 5px;
border-color:rgb(149, 145, 143);</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_3">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>220</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(139, 125, 123);
border-style: outset;
border-radius:5px; 
border-width: 5px;
border-color:rgb(149, 145, 143);</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_1">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>300</y>
     <width>360</width>
     <height>440</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 225, 156);
border-style: outset;
font-size:18px;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Comic Sans MS'; font-size:18px; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_2">
   <property name="geometry">
    <rect>
     <x>1115</x>
     <y>300</y>
     <width>360</width>
     <height>440</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 225, 156);
border-style: outset;
font-size:18px;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Comic Sans MS'; font-size:18px; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 225, 156);
border-style: outset;
font-size:18px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>70</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB Demi</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 225, 156);
border-style: outset;
border-radius:5px; 
border-width: 5px;
border-color:rgb(200, 185, 116);
font-size:20px;
font:comic-sans</string>
   </property>
   <property name="text">
    <string>ENCRYPT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>190</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB Demi</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(139, 125, 123);
border-style: outset;
border-radius:5px; 
border-width: 5px;
border-color:rgb(149, 145, 163);
font-size:20px;
</string>
   </property>
   <property name="text">
    <string>FIX</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>190</y>
     <width>90</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB Demi</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(139, 125, 123);
border-style: outset;
border-radius:5px; 
border-width: 5px;
border-color:rgb(169, 145, 143);
font-size:20px;</string>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="rot_comb_1">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>160</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell Condensed</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(139, 125, 123);
border-style: outset;
border-radius:5px; 
border-width: 5px;
border-color:rgb(149, 145, 143);
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="display_1">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>160</y>
     <width>44</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(139, 125, 123);
border-style: outset;
border-width: 5px;
border-radius: 4px;
border-color:rgb(149, 145, 143);</string>
   </property>
   <property name="text">
    <string>â… </string>
   </property>
  </widget>
  <widget class="QSpinBox" name="rot_comb_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>160</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell Condensed</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(139, 125, 123);
border-style: outset;
border-radius:5px; 
border-width: 5px;
border-color:rgb(149, 145, 143);
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="display_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>160</y>
     <width>44</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(139, 125, 123);
border-style: outset;
border-width: 5px;
border-radius: 4px;
border-color:rgb(149, 145, 143);</string>
   </property>
   <property name="text">
    <string>â… </string>
   </property>
  </widget>
  <widget class="QSpinBox" name="rot_comb_3">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>160</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell Condensed</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(139, 125, 123);
border-style: outset;
border-radius:5px; 
border-width: 5px;
border-color:rgb(149, 145, 143);
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="display_3">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>160</y>
     <width>44</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(139, 125, 123);
border-style: outset;
border-width: 5px;
border-radius: 4px;
border-color:rgb(149, 145, 143);</string>
   </property>
   <property name="text">
    <string>â… </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>549</x>
     <y>840</y>
     <width>401</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB Demi</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 225, 156);
border-style: outset;
border-radius:5px; 
border-width: 5px;
border-color:rgb(200, 185, 116);
font-size:20px;
font:comic-sans</string>
   </property>
   <property name="text">
    <string>SAVE MESSAGE</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
